pages/community/award/[name].tsx

```diff
@@ -35,9 +36,22 @@ export const getServerSideProps = compose<{ name: string }, CommitteePageProps>(
 
     if (!award) return { notFound: true, props: {} as CommitteePageProps };
 
-    const group = await new PersonnelModel().getYearGroup({ award }, [
+    const yearGroup = await new PersonnelModel().getYearGroup({ 
+      award
+      }, [
       'createdAt',
     ]);
+
+    let group: { [year: string]: any[] } = {};
+     for (const year in yearGroup) {
+       const group_unique = groupBy(yearGroup[year], "recipient" )
+       let arr_recipient = []
+       for(const recipient in group_unique){
+        if(recipient!='') arr_recipient.push(group_unique[recipient][0])
+      }
+       group[year]=arr_recipient
+     }
```

Comment:
```suggestion
import 'array-unique-proposal';

     for (const year in group)
       group[year] = group[year].uniqueBy("recipient");
```
https://github.com/tc39/proposal-array-unique

Reply:
已按要求修正